lib/ardb/adapter/base.rb

```diff
@@ -7,8 +7,8 @@ class Base
     attr_reader :schema_format, :ruby_schema_path, :sql_schema_path
 
     def initialize
-      @config_settings = Ardb.config.db_settings
-      @database = Ardb.config.db.database
+      @config_settings = Ardb.config.activerecord_connect_hash
```

Comment:
@jcredding I think I'd prefer to see the change here if it's not too much trouble.